v1/ast/compile.go

```diff
@@ -2087,6 +2093,199 @@ func (c *Compiler) checkVoidCalls() {
 	}
 }
 
+// rewriteTemplateStrings rewrites template-string calls as they appear in bodies; e.g. rules, comprehensions, etc.
+func (c *Compiler) rewriteTemplateStrings() {
+	builtinLoc := func(ref Ref) *Builtin {
```

Comment:
[nit] Perf wise, I think this would better be a standalone `func (c *Compiler) builtinLoc(ast.Ref) *Builtin`? Because right now, the closure puts stuff on the heap. But it might do that anyways, I'm just guessing a litle.